index.src.html

```diff
@@ -2343,7 +2399,11 @@
     {{InvalidStateError}}.
 2. If {{Plane/length}} is greater than |`dst.byteLength`|, throw a
     {{TypeError}}.
-3. Copy the {{Plane/[[plane buffer]]}} into <var ignore>dst</var>.
+3. Let |resource| be the [=frame resource=] refrenced by
+    {{Plane/[[parent frame]]}}'s {{VideoFrame/[[resource reference]]}}.
+4. Let |plane bytes| be the region of bytes in [=frame resource=] coresponding
+    to this plane.
+5. Copy the |plane bytes| into <var ignore>dst</var>.
```

Comment:
You should be able to drop the `ignore` if you make `dst` a `<var>` above, no?

Reply:
Nice. Done.